stable/aws-for-fluent-bit/README.md

```diff
@@ -38,9 +38,12 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `imagePullSecrets` | Docker registry pull secret | `[]` |
 | `serviceAccount.create` | Whether a new service account should be created | `true` |
 | `serviceAccount.name` | Name of the service account | `aws-for-fluent-bit` |
-| `service.extraService` | Append to existing service with this value | `""` |
-| `service.parsersFiles` | List of available parser files | `/fluent-bit/parsers/parsers.conf` |
-| `service.extraParsers` | Adding more parsers with this value | `""` |
+| `service.type`| Type of service to be created - options are ClusterIP, NodePort, LoadBalancer |`ClusterIP`|
+| `service.port`| TCP port of the serviceMonitor service. | 2020 |
+| `service.targetPort`| TCP targetPort for service to connect to fluent-bit. | 2020 |
+| `config.extraService` | Append to existing service with this value | HTTP_Server  On <br> HTTP_Listen  0.0.0.0 <br> HTTP_PORT    2020 <br> Health_Check On <br> HC_Errors_Count 5 <br> HC_Retry_Failure_Count 5 <br> HC_Period 5 |
+| `config.parsersFiles` | List of available parser files | `/fluent-bit/parsers/parsers.conf` |
+| `config.extraParsers` | Adding more parsers with this value | `""` |
```

Comment:
@razorsk8jz I think this needs to be changed? We agreed we wouldn't break backwards compatibility right? I don't see these options changed in the yaml either.